agent/tcs/handler/handler.go

```diff
@@ -134,7 +127,7 @@ func startSession(
 	}
 	seelog.Info("Connected to TCS endpoint")
 	// start a timer and listens for tcs heartbeats/acks. The timer is reset when
-	// we receive a heartbeat from the server or when a publish metrics message
+	// we receive a heartbeat from the server or when a published metrics message
```

Comment:
qq: Would we want something similar for heartbeat message?

Reply:
Don't think it is in scope of this PR, and I think we are planning to remove heartbeat message. Send you an update offline.

Reply:
makes sense. Thanks Heming.